src/test/java/org/mariadb/jdbc/BaseTest.java

```diff
@@ -528,7 +528,7 @@ public Connection createProxyConnection(Properties info) throws SQLException {
     } catch (IOException e) {
       e.printStackTrace();
     }
-    String[] splitValue = connUri.split("/");
+    String[] splitValue = connUri.split("/", 4);
```

Comment:
I don't see the point of this change, this is test code only and because if there is more slash, remaining values are joined by the next lines :  
```
 String[] subarray =
        Arrays.asList(splitValue).subList(3, splitValue.length).toArray(new String[0]);	        Arrays.asList(splitValue).subList(3, splitValue.length).toArray(new String[0]);
    String dbAndParameters = String.join("/", subarray);	    String dbAndParameters = String.join("/", subarray);
```
but sure, the whole part 
```
    String[] splitValue = connUri.split("/");
    String[] subarray =
        Arrays.asList(splitValue).subList(3, splitValue.length).toArray(new String[0]);
    String dbAndParameters = String.join("/", subarray);
```
can be simplified by 
```
    String[] splitValue = connUri.split("/", 4);
    String dbAndParameters = splitValue.length == 4 ? splitValue[3] : "";
```

Reply:
Hi @rusher ,
Yes, sorry I didn't read the updated left part carefully. It should be no problem then. I'll close the PR.